step1-07/demo/README.md

```diff
@@ -109,13 +111,13 @@ interface TodoListProps {
 }
 ```
 
-> Note that the `[]` notation does not mean an array, it is a [computed property](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Computed_property_names) notation.
+> Note that `[id: string]` does not indicate an array; it is an object [index signature](https://www.typescriptlang.org/docs/handbook/interfaces.html#indexable-types).
```

Comment:
Index signature is actually the correct term here rather than computed property (though an index signature would allow an object to use computed property names)